daemon/src/monitor.rs

```diff
@@ -120,8 +370,9 @@ impl Cfd {
                     monitor_cet_timelock: true,
                     monitor_refund_timelock: true,
                     monitor_refund_finality: true,
-                    monitor_revoked_commit_transactions: true, /* After rollover, the other party
-                                                                * might publish old states. */
+                    monitor_revoked_commit_transactions: true,
+                    /* After rollover, the other party
+                     * might publish old states. */
```

Comment:
🔧 You probably meant to put this on top of `monitor_revoked_commit_transactions` (or keep it beside it).

Reply:
That was an auto-formatting error. Will fix!